htmd/qm/psi4.py

```diff
@@ -118,7 +118,10 @@ def _command(self):
         return 'export HTMD_PSI4_WORKDIR=$(pwd)\npsi4 -i psi4.in -o psi4.out &> psi4.log'
 
     def _completed(self, directory):
-        return os.path.exists(os.path.join(directory, 'psi4.out'))
+        # Abuse "timer.dat" to detect if a Psi4 job has completed.
```

Comment:
Is it not possible to use sentinels the same way we use them in our queues? By wrapping the executable in a bash script with the sentinel?

Reply:
Ah ok I read the discussion